src/components/contextual-menu/menu-layouts/TextMenu.js

```diff
@@ -13,6 +13,15 @@ class TextMenu extends Component {
 
         this.handleFontFamilyChange = this.handleFontFamilyChange.bind(this);
         this.handleFontSizeChange = this.handleFontSizeChange.bind(this);
+        this.handleFontStyleChange = this.handleFontStyleChange.bind(this);
+        this.handleFontWeightChange = this.handleFontWeightChange.bind(this);
+        this.handleTextAlignChange = this.handleTextAlignChange.bind(this);
+        this.handleTextDecorationChange = this.handleTextDecorationChange.bind(this);
+        this.handleWidthChange = this.handleWidthChange.bind(this);
+        this.handleHeightChange = this.handleHeightChange.bind(this);
+        this.handleXChange = this.handleXChange.bind(this);
+        this.handleXChange = this.handleYChange.bind(this);
```

Comment:
Looks like you have two `handleXChanges` here, is one supposed to be for Y?

Reply:
Yes! Thanks for catching that, I'll change it